moml/__init__.py

```
@@ -32,13 +32,48 @@
-from . import core
-from . import data
-from . import models
-from . import pipeline
-from . import simulation
-from . import utils
+# Subpackages with conditional imports
+try:
+    from . import core
+except ImportError:
+    import warnings
+    warnings.warn(f"Could not import core module")
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)